scanner/rails.go

```diff
@@ -332,24 +320,24 @@ func RailsCallback(appName string, srcInfo *SourceInfo, plan *plan.LaunchPlan, f
 			if pendingError != nil {
 				pendingError = errors.Wrap(pendingError, "Failed to add dockerfile-rails gem")
 			} else {
-				cmd = exec.Command(bundle, "install", "--quiet")
-				cmd.Stdin = nil
-				cmd.Stdout = os.Stdout
-				cmd.Stderr = os.Stderr
-
-				pendingError = cmd.Run()
-				if pendingError != nil {
-					pendingError = errors.Wrap(pendingError, "Failed to install dockerfile-rails gem")
-				} else {
-					generatorInstalled = true
-				}
+				generatorInstalled = true
 			}
 		}
 	} else {
 		// proceed using the already installed gem
 		generatorInstalled = true
 	}
 
+	cmd := exec.Command(bundle, "install", "--quiet")
+	cmd.Stdin = nil
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	err = cmd.Run()
+	if err != nil {
+		return errors.Wrap(pendingError, "Failed to install bundle, exiting")
```

Comment:
Is it okay to throw away `err`? The original code is wrapping the error from `cmd.Run()`.